app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Base class for vaccine errors."""
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    """Raised when visitor has no vaccine."""
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    """Raised when vaccine has been expired."""
+    pass

```

Comment:
For `OutdatedVaccineError` class, you should provide a descriptive error message. For example:

```python
class OutdatedVaccineError(VaccineError):
    def __init__(self):
        super().__init__('The vaccine is outdated.')
```